openedx/features/edly/api/serializers.py

```diff
@@ -49,3 +50,30 @@ def get_mobile_enabled(self, obj):  # pylint: disable=unused-argument
         Returns mobile_enabled flag
         """
         return self.context['site_configuration'].get('MOBILE_ENABLED', False)
+
+
+class EdlySubOrgSerializer(serializers.ModelSerializer):
+    """
+    Serializer for edly sub_organization model
+    """
+     
+    class Meta:
+        """
+        Meta attribute for the EdlySubOrganization Model 
+        """
+        model= EdlySubOrganization
+        fields = ['name', 'slug']
+
+
+class MutiSiteAccessSerializer(serializers.ModelSerializer):
+    """
+    Serializer for Mutisite access endpoint
+    """
+    sub_org = EdlySubOrgSerializer(read_only=True)
+
+    class Meta:
+        """
+        Meta attribute for the MutiSiteAccess Model
+        """
+        model = EdlyMultiSiteAccess
+        fields = ['id', 'sub_org']
```

Comment:
we can simplify this into one single serializer as edly_sub_org is already present in edly_multisite_access model we can directly fetch those field values

Reply:
resolved